src/eval-frame/components/reps/react-table-styles.css

```diff
@@ -0,0 +1,46 @@
+.ReactTable {
+  font-family: "Open Sans", sans-serif;
+}
+
+.ReactTable .pagination-top {
+  padding: 2px;
+}
+
+.ReactTable .-pagination {
+  padding: 2px;
+  box-shadow: none;
+  border: none;
+}
+
+.ReactTable .-pagination .-pageSizeOptions {
+  margin: 0px 10px;
+}
+
```

Comment:
adding this style might help others realize they can click a value and get the detailed rep:
```css
div.rt-td {
    cursor: pointer;
}
```

Reply:
:+1: